autoscaler/notification.py

```diff
@@ -152,47 +144,14 @@ def notify_drained_node(self, node, pods):
 
         message = 'Node {} drained.'.format(node)
         message += '\n'
-        message += 'Pod affected: {}'.format(pods_string)
+        # message += 'Pod affected: {}'.format(pods_string)
```

Comment:
I was thinking that : 
``` 
Node k8s-agentpool1-33475507-1 drained.
 ```
was enough information, instead of : 
``` 
Node k8s-agentpool1-33475507-1 drained.
Pod affected: kube-system/kube-proxy-2tn0d
```

If you want to log the pods, I will remove the comment.
Let me know